pymc/distributions/timeseries.py

```diff
@@ -967,55 +998,6 @@ def dist(cls, dt, sde_fn, sde_pars, *, init_dist=None, steps=None, **kwargs):
 
         return super().dist([init_dist, steps, sde_pars, dt, sde_fn], **kwargs)
 
-    @classmethod
-    def rv_op(cls, init_dist, steps, sde_pars, dt, sde_fn, size=None):
-        # Init dist should have shape (*size,)
-        if size is not None:
-            batch_size = size
-        else:
-            batch_size = pt.broadcast_shape(*sde_pars, init_dist)
-        init_dist = change_dist_size(init_dist, batch_size)
-
-        # Create OpFromGraph representing random draws from SDE process
-        # Variables with underscore suffix are dummy inputs into the OpFromGraph
-        init_ = init_dist.type()
-        sde_pars_ = [x.type() for x in sde_pars]
-        steps_ = steps.type()
```

Comment:
Is this `type` thing not required anymore? I remember having to do that when I wrote distributions.

Reply:
That was all a side effect of not being able to use shared variables and constants as inputs. No longer needed